sources/fips.json

```diff
@@ -3,7 +3,10 @@
 	"fullname": "Federal Information Processing Standards",
 	"name": "fips",
 	"prefix": "fips",
-    	"key": "code",
+	"key": "code",
 	"url": "http://www.nist.gov/itl/fips.cfm",
-	"license": "https://www.usa.gov/government-works"
-}
+	"license": "https://www.usa.gov/government-works",
+	"license_type": "CC0",
+	"license_text": "Most U.S. government creative works such as writing or images are copyright-free.",
```

Comment:
The previous paragraph text on that link is better:

`United States government creative works, including writing, images, and computer code, are usually prepared by officers or employees of the United States government as part of their official duties. A government work is generally not subject to copyright in the United States and there is generally no copyright restriction on reproduction, derivative works, distribution, performance, or display of a government work.`